### ISSUE TITLE
Use Monospaced font for UI

### ISSUE THREAD

**Issue by NghiaTranUIT** [ProxymanApp/Proxyman#1857]
## Description
Some users request that it should be nice to have a monospaced font for the entire UI app. It should apply to:
- The Header, Query, Auth, ... tab
- The Body
- The Raw Tab
- Main Table View

## Why this feature/change is important?
- Easier to read text

**Comment by NghiaTranUIT** [comment]
Beta: https://download.proxyman.io/beta/Proxyman_4.14.0_Support_Monospaced_Font.dmg

![CleanShot 2023-11-27 at 20 17 47@2x](https://github.com/ProxymanApp/Proxyman/assets/5878421/aec0d961-5b9f-4e0e-88ab-9f78e9124237)

**Comment by T0sng** [comment]
I think there should be an option for users to customize the monospace font. I like using a monospace font, but I don't like the one in this version. I hope there can be an option to change it to another monospace font :) Thank you.

**Comment by T0sng** [comment]
> @T0sng it's the official monospaced SF font from Apple. I believe that it fits the app well.它是 Apple 的官方等宽 SF 字体。我相信它非常适合该应用程序。
Yes, I understand that he looks very good in an English language environment, but in a Chinese environment, especially with some complex characters, it looks very bad.
